src/pretix/base/models/event.py

```diff
@@ -330,6 +330,21 @@ def get_payment_providers(self) -> dict:
                 providers[pp.identifier] = pp
         return providers
 
+
+    @property
+    def get_event_microdata(self):
```

Comment:
Drop the ``@property`` or name it to ``event_microdata``. I don't really care, both are equally fine, but a property should never be named with a verb like ``get_…``.